cypress/e2e/hub/collections-detail.cy.ts

```diff
@@ -186,65 +191,69 @@ describe('Collections Details', () => {
     });
   });
 
-  it('can sign a selected version of a collection', () => {
+  it.skip('can sign a selected version of a collection', () => {
```

Comment:
the selected version changes from 1.0.0 to 1.2.3 when clicking the sign-version kebab button